rest-model/src/main/java/org/jboss/pnc/rest/restmodel/ProductMilestoneRest.java

```diff
@@ -41,6 +42,7 @@
     private Integer id;
 
     @NotNull(groups = {WhenCreatingNew.class, WhenUpdating.class})
+    @Pattern(regexp = "^[0-9]+\\.[0-9]+(\\.[0-9]+)?\\.[\\w]+$", message = "Version doesn't match the required pattern ^[0-9]+\\.[0-9]+(\\.[0-9]+)?\\.[\\w]+$")
```

Comment:
Can you describe in a comment what this regex does as they are hard to read.

Reply:
OK